source/includes/steps-create-data-encryption-key.yaml

```diff
@@ -0,0 +1,120 @@
+title: Read the local master key from a file
+ref: read-local-master-key-from-file
+level: 4
+content: |
+  First, read the file containing the local master key, generated in the
+  :ref:`Create a Master Key <fle-create-a-master-key>` section:
```

Comment:
nit: the local master key _that you_ generated

Reply:
Thanks, that sounds better, will fix. Just for my own understanding, is "generated in..." an example of a non-restrictive clause? And we prefer to use which/that over commas? Or am I completely wrong?

Reply:
Whoops, lost my comment here (created a review instead of single comment, and then subsequently discarded the review). I prefer your version, but the question was just about whether "generated in the..." is an example of a non-restrictive clause and that we prefer that/which over commas, or whether it is not even a non-restrictive clause.

Reply:
AFAIK non-restrictive clause just means that the clause does not increase specificity. I think this might qualify under that definition but I'm not certain.

I think the main way we use it is in appositives, but I don't know if we use verb phrase appositives anywhere. I personally don't prefer this style because it feels like passive voice, i.e.:

"read the file containing the local master key (that was) generated in the Create a Master Key section (by whom?)."

Reply:
Thanks for the detailed explanation!